### PR TITLE
FOKUS changeset

### PR DESCRIPTION
**Pull request by thanosa75**
@majidsalehighamsari please review collected changes for this merge request

### CODE REVIEW

**Comment by thanosa75** [comment]
Great, thanks

File: deployment/docker/applications/intelmq/Dockerfile [hunk 1]
```diff
@@ -28,11 +28,14 @@ RUN pip3 install . && \
...
  mkdir -p /opt/intelmq/var/log && \
- chmod -R 0777 /opt/intelmq && \
+#review chmod -R 775 /opt/intelmq && \
  chown -R intelmq:intelmq /opt/intelmq
 
-RUN git clone https://github.com/certtools/intelmq-manager.git /tmp/intelmq-manager && \
- cp -R /tmp/intelmq-manager/intelmq-manager/* /var/www/html/ && \
+RUN git clone https://github.com/certtools/intelmq-manager.git /tmp/intelmq-manager
+WORKDIR /tmp/intelmq-manager
+RUN git checkout tags/1.1.0
```
Comment (wagner-certat) [inline deployment/docker/applications/intelmq/Dockerfile]: Here you are checking out 1.1.0 and above you check out 1.1.1 *confused*

File: deployment/docker/applications/intelmq/Dockerfile [hunk 2]
```diff
@@ -12,12 +12,12 @@ RUN apt-get update &&   \
...
  apt-get install -y sudo build-essential libcurl4-gnutls-dev libgnutls28-dev python3-dev && \
 # apt-get install -y nano vim && \
- rm -rf /var/lib/apt/lists/*
+ rm -rf /var/lib/apt/lists/*checkout
 
 RUN git clone https://github.com/certtools/intelmq.git /opt/dev_intelmq
 WORKDIR /opt/dev_intelmq
-#RUN git checkout tags/1.1.0rc1
-RUN git checkout
+RUN git checkout tags/1.1.1
```
Comment (wagner-certat) [inline deployment/docker/applications/intelmq/Dockerfile]: Is there an incompatibility with 2.0.0 or is there another reason you are not using the current version?

**Comment by majidsalehighamsari** [comment]
2.0.0 is new, we did not test and use this version.

**Pull request merged**